src/App.jsx

```diff
@@ -14,45 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
consider to make one constant as  object SORT_FIELD instead 2 const as strings

```suggestion
const SORT_FIELD = {
  LENGTH: 'length',
  NAME: 'name,
}
```